common/wonders/wc_wonders_historical.txt

```diff
@@ -621,3 +620,105 @@ wonder_ulduar = {
 		}
 	}
 }
+
+#Janeiro Point
+wonder_janeiro_point = {
+	allow_construction = no
+	picture = GFX_wonder_strip_janeiro_point
+	potential = {
+	}
+	active = {
+	}
+	flags = {
+		is_historical
+		cost_medium
```

Comment:
Why it's medium? The first stage of `wonder_statue_ruler` costs `1500` and it's considered `cost_low`.